src/JupyterLibrary/clients/jupyterlab/Output.robot

```diff
@@ -39,3 +39,16 @@ Screenshot Each Output of Active JupyterLab Document
         Screenshot Each Output of Active JupyterLab Cell    ${prefix}_cell_${i}
         Screenshot Markdown of Active JupyterLab Cell    ${prefix}_cell_${i}
     END
+
+Wait Until a Given JupyterLab Code Cell Is Not Active In a Given Tab
+  [Documentation]  Waits until the given cell in a specific JL tab no longer has an active prompt "[*]:".
+  ...              The cell index starts from 1 and you must count empty and markdown cells too.
+  [Arguments]  ${tab_id_to_wait}  ${cell_n}  ${timeout}=120seconds
+  Wait Until Element Is Not Visible  xpath=//div[@aria-labelledby="${tab_id_to_wait}"]/${NB TAB XP CONTENT}/div[${cell_n}]/${CELL XP INPUT STATUS ICON}   ${timeout}
```

Comment:
Many of these `aria-label` attributes were only added very recently, and won't work with the older versions of lab under test. This is _maybe_ fine, but it's worth exploring some alternate approaches to finding them.

Also, the 2 minutes is _ridiculously_ long as a default. I guess it's all relative anyway but otherwise the longest one is `30s`, and I'd recommend adopting that one as a default.

Out of scope of this: we should see whether SeleniumLibrary already has some related ones, and if so, use them (perhaps with some ghetto math). Otherwise, we could name and categorize our own defaults, e.g. `${RFJL DEFAULT WAIT LONG}`, `${RFJL DEFAULT WAIT SHORT}` that could be easily overriden at the suite level (or updated from the CLI when retrying).

Reply:
> retro-compatibility


I tried this keyword with JL 3.2.5, 2.3.2 and 1.2.21. I see that it works for both v3 and v2. It's not working for v1. Will update the PR description to include the compatibilities.

> timeout


agree with you. I've updated it to 30 seconds.

> Out of scope of this: we should see whether SeleniumLibrary already has some related ones [...]


agree. We could investigate this